show/main.py

```diff
@@ -228,6 +228,35 @@ def alias(interfacename):
 
     click.echo(tabulate(body, header))
 
+# 'Neighbor MetaData' subcommand ("show interfaces neighbor")
```

Comment:
This comment should simply notate the expected command that this function creates, like

`# 'neighbor' subcommand ("show interfaces neighbor")`